src/main/java/net/pms/dlna/protocolinfo/PanasonicDmpProfiles.java

```diff
@@ -0,0 +1,526 @@
+	 * linked {@link DeviceProtocolInfo} instance. This method assumes that the
+	 * renderer sends the same string every time.
+	 *
+	 * @param dmpProfilesString the {@code X-PANASONIC-DMP-Profile} string to
+	 *            parse.
+	 * @param renderer the {@link RendererConfiguration} for which to apply the
+	 *            parsing results.
+	 * @throws IllegalStateException If {@code renderer}'s
+	 *             {@code deviceProtocolInfo} is {@code null}.
+	 */
+	public static void parsePanasonicDmpProfiles(String dmpProfilesString, RendererConfiguration renderer) {
+		if (renderer == null) {
+			return;
+		}
+		if (renderer.deviceProtocolInfo == null) {
+			throw new IllegalStateException(
+				"Panasonic DMP profiles cannot be parsed before the renderer's deviceProtocolInfo is instantiated"
+			);
+		}
+		boolean added = false;
+		if (renderer.panasonicDmpProfiles == null) {
+			renderer.panasonicDmpProfiles = new PanasonicDmpProfiles(renderer.deviceProtocolInfo, dmpProfilesString);
+			added = true;
+		} else if (!renderer.panasonicDmpProfiles.isPopulated()) {
+			added = renderer.panasonicDmpProfiles.add(dmpProfilesString);
+		}
+		if (added && LOGGER.isTraceEnabled() && !renderer.panasonicDmpProfiles.isEmpty()) {
+			LOGGER.trace(
+				"Received X-PANASONIC-DMP-Profiles from \"{}\":\n\n{}",
+				renderer.getConfName(),
+				renderer.panasonicDmpProfiles.toString(true)
+			);
+		}
+	}
+
+	/**
+	 * Tries to convert an element from a {@code X-PANASONIC-DMP-Profile} string
+	 * to a {@link ProtocolInfo} instance. This is a manual mapping which must
+	 * be extended when new profiles are discovered.
+	 *
+	 * @param dmpProfile the {@code X-PANASONIC-DMP-Profile} string element.
+	 * @return A new {@link ProtocolInfo} instance if one could be created, or
+	 *         {@code null}.
+	 * @throws ParseException If there is a problem while parsing
+	 *             {@code dmpProfile}.
+	 */
+	public static ProtocolInfo dmpProfileToProtocolInfo(String dmpProfile) throws ParseException {
+		if (isBlank(dmpProfile)) {
+			return null;
+		}
+		dmpProfile = dmpProfile.trim();
+		ProtocolInfoAttribute attribute = DLNAOrgProfileName.FACTORY.getProfileName(dmpProfile);
+		if (attribute != null) {
+			// Mime-types must be mapped manually, as this information is missing from X-PANASONIC-DMP-Profile
+			MimeType mimeType;
+			if (attribute.getValue().startsWith("JPEG")) {
+				mimeType = new MimeType("image", "jpeg");
+			} else if (attribute.getValue().startsWith("PNG")) {
+				mimeType = new MimeType("image", "png");
+			} else if (attribute.getValue().startsWith("GIF")) {
+				mimeType = new MimeType("image", "gif");
+			} else if (attribute.getValue().startsWith("MPEG")) {
+				mimeType = new MimeType("video", "mpeg");
+			} else if (attribute.getValue().startsWith("AC3")) {
+				mimeType = new MimeType("audio", "vnd.dolby.dd-raw");
+			} else if (attribute.getValue().startsWith("AMR")) {
+				mimeType = new MimeType("audio", "mp4");
```

Comment:
AMR_3GPP and AMR_WBplus ==> audio/3gpp
Mandatory support for 3GP profile, but if 3GP is very popular profile, AMR support is not on Panasonic, as you say, but i think remember that you've it on some Onkyo and others renderers of course.
My point was that 3gpp is much more popular, wide spreaded.